src/models/User.ts

```diff
@@ -0,0 +1,16 @@
+import mongoose, { Schema, Document } from "mongoose";
+
+export interface IUser extends Document {
+  githubId: string;
```

Comment:
What if we are working with non-developer which use google to authorize himself?

Reply:
As fat as i know. For now we chose using only github auth.

Reply:
Not exactly, we are using github for developers and google for testers/UI/UX/ and rest of users who aren't dev